IsraelHiking.Web/src/index.html

```diff
@@ -7,6 +7,7 @@
     <meta property="og:site_name" content="IsraelHiking.OSM.org.il" />
     <meta property="og:url" content="https://israelhiking.osm.org.il/" />
     <meta property="og:type" content="website" />
+    <!-- IHM START -->
```

Comment:
I was thinking of using search and replace (in the .net code) instead of defining a section that will be replaced as a section is more "fragile".

Reply:
I agree.... rendering the whole head as a template makes sense here... However, this works OK now, assuming we still want to use the index.html built using node.